server/utils/passport/local-login.js

```diff
@@ -20,28 +24,24 @@ export default (req, res, next, user) => {
   })
   //If the user exists, serialize the user to the session
   req.login(user, function (err) {
+    if (err) return res.redirect(`${basePath}/login?e=${err}`)
```

Comment:
we should use route constants for these.

```suggestion
    if (err) return res.redirect(`${routes.login}?e=${err}`)
```

Reply:
[de48040](https://github.com/AMP-SCZ/dpdash/pull/293/commits/de48040cb5b39f723d4ad1877be815b064027992)